### PR TITLE
Chore: Move service definitions to core services package

### PR DESCRIPTION
**Pull request by sampaiodiego**
## Issue(s)

ARCH-118

## Further comments

There are a few things missing:

- [x] Pass `StreamerCentral` to `LocalBroker` via dependency injection
- [x] Remove `api` export from `apps/meteor/server/sdk/api.ts`
- [x] `getRoutingManagerConfig` was removed, check if it was really not being used
- [x] `packages/core-sdk/src/types/IVoipService.ts` check how `getConnector()` is used and figure how to fix it out

### CODE REVIEW

File: packages/core-services/src/lib/asyncLocalStorage.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import type { IServiceContext } from '../types/ServiceClass';
+import { FibersContextStore } from './ContextStore';
+
+// TODO Evalute again using AsyncContextStore instead of FibersContextStore in a future Meteor release (after 2.5)
```
Comment (KevLehman) [inline packages/core-services/src/lib/asyncLocalStorage.ts:4]: Aren't we in meteor 2.6? :trollface:

Comment (sampaiodiego) [reply packages/core-services/src/lib/asyncLocalStorage.ts:4]: true.. not sure why Meteor 2.5 though, the bump to Node 14 was on Meteor 2.3 .. I'd say we should re-evaluate this when Meteor updates to Node 16, that's when Fibers will be removed from Meteor.

File: apps/meteor/app/apps/server/bridges/messages.ts [hunk 2]
```diff
@@ -4,11 +4,11 @@ import type { IMessage } from '@rocket.chat/apps-engine/definition/messages';
 import type { IUser } from '@rocket.chat/apps-engine/definition/users';
 import type { IRoom } from '@rocket.chat/apps-engine/definition/rooms';
 import type { ISubscription } from '@rocket.chat/core-typings';
+import { api } from '@rocket.chat/core-services';
```
[the remark this answers is not in the document]
Comment (sampaiodiego) [reply apps/meteor/app/apps/server/bridges/messages.ts:7]: 🤔 interesting

File: _templates/service/new/service.ejs.t [hunk 3]
```diff
@@ -3,8 +3,8 @@ to: ee/apps/<%= name %>/src/service.ts
 ---
 import type { Document } from 'mongodb';
 import polka from 'polka';
+import { api } from '@rocket.chat/core-services';
```
[the remark this answers is not in the document]
Comment (sampaiodiego) [reply _templates/service/new/service.ejs.t:6]: yep.. definitely..

File: packages/core-services/src/lib/asyncLocalStorage.ts (hunk 1 above)
Comment (KevLehman) [reply packages/core-services/src/lib/asyncLocalStorage.ts:4]: Noice

**Review by ggazzo — APPROVED** [review]

**Pull request merged**